images/cross-cloud/sle-hpc/ondemand/15-sp7/preferences.yaml

```diff
@@ -0,0 +1,22 @@
+image:
+  preferences:
+    - _include:
+        - base/common
+    - _attributes:
+        profiles: [Azure]
+        arch: x86_64
+      _include:
+        - csp/azure/settings/hpc/ondemand
+    - _attributes:
+        profiles: [Azure]
+        arch: aarch64
+      _include:
+        - csp/azure/settings/aarch64
+        - csp/azure/settings/hpc/ondemand
+    - _attributes:
+        profiles: [EC2]
+      _include:
+        - csp/ec2/settings/hpc/ondemand
+    - _attributes:
+        profiles: [GCE]
+      _include: csp/gce/settings/hpc/ondemand
```

Comment:
Not 100% sure about the syntax but does the value `csp/gce/settings/hpc/ondemand` need to be a bullet point on a new line?

Reply:
> Not 100% sure about the syntax but does the value `csp/gce/settings/hpc/ondemand` need to be a bullet point on a new line?

Keg supports a single string with `_include` too, but since we generally use the list format I'll change it for consistency.